solarforecastarbiter/io/reference_observations/tests/conftest.py

```diff
@@ -53,6 +53,17 @@ def site_dicts():
                                  "network": 'NREL MIDC',
                                  "network_api_id": 'BMS',
                                  "network_api_abbreviation": 'abbrv'},
+        },
+        {
+            'name': 'site4',
+            'latitude': 4,
+            'longitude': -5,
+            'elevation': 12,
+            'timezone': 'Etc/GMT+8',
+            'extra_parameters': {"observation_interval_length": 1,
+                                 "network": 'Unincorportated',
```

Comment:
```suggestion
                                 "network": 'Unincorporated',
```

Reply:
what is this anyways?

Reply:
Just ensuring it doesn't try to jam some reference site it doesn't know how to gather data for into the database.